integrations/hardhat/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Change Log
+
+All notables changes to this project are documented in this file.
+
+The format is inspired by [Keep a Changelog].
+
+[Keep a Changelog]: https://keepachangelog.com/en/1.0.0/
+
+## 2.22.2-next.0 (2024-08)
```

Comment:
I guess `next` is a direct replacement for `alpha` then.

Reply:
Sorry I had to investigate this a bit, so we don't need to specify this in package.json.

We can just rely on tags, and keep the version in package.json as e.g. `v2.22.2`

Reply:
I'm going to stick with this for now, as it's part of the semver spec and has specific meaning for prereleases

https://www.npmjs.com/package/semver#user-content-prerelease-tags